src/libs/SidebarUtils.ts

```diff
@@ -117,12 +124,22 @@ function getOrderedReportIDs(
     betas: Beta[],
     policies: Record<string, Policy>,
     priorityMode: ValueOf<typeof CONST.PRIORITY_MODE>,
-    allReportActions: OnyxCollection<ReportAction[]>,
+    allReportActions: OnyxCollection<ReportActions>,
+    transactionViolations: TransactionViolations,
 ): string[] {
+    const reportActionsCount: ReportActionsCount | null =
+        allReportActions &&
+        Object.entries(allReportActions).reduce<ReportActionsCount>((acc, [reportID, reportActions]) => {
+            if (reportActions) {
+                acc[reportID] = {reportActionsCount: Object.keys(reportActions as Record<string, ReportAction>).length};
+            }
+            return acc;
+        }, {});
```

Comment:
Cramming all of this into a single statement makes it hard to read, can we update like so?
```suggestion
    const reportActionsCount: ReportActionsCount | null = null;
    if (allReportActions) {
        Object.entries(allReportActions).reduce<ReportActionsCount>((acc, [reportID, reportActions]) => {
            acc[reportID] = _.size(reportActions);
            return acc;
        }, {});
    }
```

Reply:
I broke this out into separate functions to try and make it more readable. Let me know if that's good!